types/use-analytics/index.d.ts

```diff
@@ -1,32 +1,33 @@
 // Type definitions for use-analytics 0.0
-// Project: https://github.com/DavidWells/analytics/tree/master/packages/use-analytics
-// Definitions by: Harry Cruse <https://github.com/CruseCtrl>
+// Project: https://github.com/DavidWells/analytics
```

Comment:
I think it's better to leave this pointing to the specific location of `use-analytics`, rather than the general location of `analytics` which also includes other packages